src-web/tableDefinitions/utils.js

```diff
@@ -136,6 +135,9 @@ function pushRows(items, rows, def, locale) {
           title: refetch ? key.transformFunction(item, locale, refetch) : key.transformFunction(item, locale),
           rawData: value
         }
+        if (key.textFunction && typeof key.textFunction === 'function') {
+          value.text = key.textFunction(item, locale)
+        }
```

Comment:
Nice. I originally tried to capture strings via the `rawData` key, but adding a `textFunction` is probably clearer and more extensible.